src/Marten.Testing/Documents/Target.cs

```diff
@@ -119,14 +120,15 @@ public Target()
 
         public bool Flag { get; set; }
 
+        [JsonInclude]
         public string StringField;
 
         public double Double { get; set; }
         public decimal Decimal { get; set; }
         public DateTime Date { get; set; }
         public DateTimeOffset DateOffset { get; set; }
 
-        public float Float;
+        public float Float { get; set; }
```

Comment:
There's purposely a mix of properties and public fields to ensure that Marten works with both. If we do this, we just need to make sure there are tests specifically for Newtonsoft against public fields. So, ability to use a field as an identity and Linq querying.

Annoying, I know.

Reply:
System.Text.Json requires explicit attribute on fields. I reverted the change and added attribute with a comment informing that's only needed for it.